flink-python/pyflink/table/table_environment.py

```diff
@@ -659,6 +660,32 @@ def list_temporary_views(self) -> List[str]:
         j_view_name_array = self._j_tenv.listTemporaryViews()
         return [item for item in j_view_name_array]
 
+    def list_models(self) -> List[str]:
+        """
+        Gets the names of all model available in the current namespace (the current
```

Comment:
nit: does this include temporary models. 
In terms of wording, what is an unregistered model?